Dockerfile

```
@@ -58,36 +45,38 @@ RUN apt-get install -y \
     libidn11-dev \
     libicu-dev \
     libjemalloc-dev \
-    tini
-
-RUN corepack enable ; \
-    yarn set version classic
+    tini ; \
+    apt-get autoremove -y ; \
+    apt-get clean ; \
```

```suggestion
    apt-get clean ; \
    rm -rf /var/lib/apt/lists/* ; \
```

You can also remove the lists to save some space.